docs/fundamentals/write-operations.txt

```diff
@@ -258,6 +258,85 @@ An **upsert** operation lets you perform an update or insert in a single
 operation. This operation streamlines the task of updating a document or
 inserting one if it does not exist.
 
+Starting in v4.7, you can perform an upsert operation by using either of
+the following methods:
+
+- Use the ``upsert()`` method. When you use this method,
+  you can perform a **batch upsert** to change or insert multiple documents
+  in one operation.
+
+- Use the ``update()`` method and specify the ``upsert``
```

Comment:
S: since you introduce this list with "using either of the following methods," I think it makes sense to start each bullet with the method itself. Or, update that intro text to something like "in the following ways:"
```suggestion
- ``upsert()``: When you use this method,
  you can perform a **batch upsert** to change or insert multiple documents
  in one operation.

- ``update()``: When you use this method, specify the ``upsert``
```

Reply:
Applies to the query builder page as well